lib/linguist/languages.yml

```diff
@@ -3429,6 +3429,18 @@ Nemerle:
   tm_scope: source.nemerle
   ace_mode: text
   language_id: 243
+NEO:
+  type: programming
+  extensions:
+  - ".neo"
+  aliases:
+  - neo
+  interpreters:
+  - neo
+  color: "#3572A5"
+  ace_mode: text
+  tm_scope: none
+  language_id: 8080
```

Comment:
Please follow the guidelines and use the scripts provided to update `languages.yml`.

Reply:
ok